### PR TITLE
Add French (fr_FR) translation

### PR DESCRIPTION
**Pull request by MattMarcha**
Completion of the fr_FR translation. This is a first draft, to be reworked when the beta version is released.

### CODE REVIEW

**Comment by lah7** [comment]
Thank you! I'll keep the pull request open and will merge before a beta release, just so others considering `fr_FR` know it is being worked on.

File: source/launchers/polychromatic.desktop [hunk 1]
```diff
@@ -3,7 +3,9 @@ Type=Application
 Version=1.0
 Name=Polychromatic
 GenericName=Device Manager for RGB Lighting
-Comment=Configure connected lighting peripherals
+GenericName[fr]=Gestionnaire d'éclairage RGB des périphériques
+Comment=Configures connected lighting peripherals
```
Comment (lah7) [inline source/launchers/polychromatic.desktop:7]: For `Comment`, the [Desktop Entry spec](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#recognized-keys) doesn't standardise plural vs non-plural. Considering other entries on the system and the specification's example, non-plural is the most common for English.

Please revert this line back to `Configure`.

Comment (MattMarcha) [reply source/launchers/polychromatic.desktop]: Oops ! I kept the wrong line when I rebased, sorry. It is fixed.

**Comment by lah7** [comment]
A new release (v0.7.0) is coming up! 👏🏼  There should be no more string changes now.

I introduced a minor conflict in `.desktop` by commenting out some features that won't make it for this new release.

---
Locally, I was testing a rebase of your `fr_FR` against the latest (180+) changes and noticed some bugs.

image

It looks like a bug in the software or in the scripts that generate locales. I'll look into getting them fixed in the meantime.

I'd hate for you to duplicate efforts, but it seems that many unchanged strings (like "Refresh") haven't merged with your previous work. I apologize that the scripts lack some refinement there. It could be because I added comments to help identify them. I'll need to look into that ASAP.

**Comment by lah7** [comment]
> [...] it seems that many unchanged strings (like "Refresh") haven't merged with your previous work.

Hmm, it appears to be okay now. It's showing as 71% translated (as opposed to a low number like 5% yesterday). Phew! 😅

**Comment by MattMarcha** [comment]
Thank you for the update !
My Razer Blade is currently under repair. I'll check fr translation for the new release as soon as possible, probably in a few weeks (~2).

**Comment by lah7** [comment]
A new release is imminent. I fixed the conflict and merged the translation so far. Thanks again!

**Pull request merged**